Detectors/ITSMFT/common/reconstruction/include/ITSMFTReconstruction/DecodingStat.h

```diff
@@ -48,8 +48,8 @@ struct ChipStat {
     UnknownWord,                  // Unknown word was seen
     RepeatingPixel,               // Same pixel fired more than once
     WrongRow,                     // Non-existing row decoded
-    APE_STRIP_START,              // lane entering strip data mode | See https://alice.its.cern.ch/jira/browse/O2-1717
-    APE_STRIP_STOP,               // lane exiting strip data mode
+    APE_STRIP,                    // lane data stripped for this chip event (behaviour changed with RU FW v1.16.0, for general APE behaviour see  https://alice.its.cern.ch/jira/browse/O2-1717)
+    APE_RESERVED_F3,              // reserved F3
     APE_DET_TIMEOUT,              // detector timeout (FATAL)
```

Comment:
You should also change `APE_STRIP_START` to `APE_STRIP` at  https://github.com/AliceO2Group/AliceO2/blob/7dd6ae1083a328ca8663197e94cd760b88e0d7d6/Detectors/ITSMFT/common/reconstruction/include/ITSMFTReconstruction/DecodingStat.h#L153 and https://github.com/AliceO2Group/AliceO2/blob/7dd6ae1083a328ca8663197e94cd760b88e0d7d6/Detectors/ITSMFT/common/reconstruction/include/ITSMFTReconstruction/DecodingStat.h#L166

Reply:
Hi @shahor02,
Thanks a lot for spotting this. I've pushed the changes.